packages/paste-website/src/hooks/useWindowSize.tsx

```diff
@@ -41,8 +41,8 @@ function useWindowSize(): WindowSizeState {
           breakpointIndex: getBreakpointIndex(window.innerWidth),
         });
       },
-      100,
-      {leading: false, trailing: true, maxWait: 400}
+      50,
+      {leading: false, trailing: true, maxWait: 150}
```

Comment:
This was too slow before, these new numbers still help performance while being less jarring.